sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -29,6 +29,8 @@ public partial class CertificateClientLiveTests : CertificatesTestBase
     {
         public CertificateClientLiveTests(bool isAsync, CertificateClientOptions.ServiceVersion serviceVersion) : base(isAsync, serviceVersion)
         {
+            // TODO: https://github.com/Azure/azure-sdk-for-net/issues/11634
```

Comment:
The Key Vault body differences aren't really something we can solve since we're generating keys on the server (will always be different). We could write our own matcher, but I imagine this problem will be common. Could we instead consider an approach that allows us to specify what should be ignored. For example, for JSON  (most of our body content across the clients) we could specify JSONPaths. A comparer could do a recursive JSON object comparison and ignore anything specified by the JSONPath (or some filter). Idea for a future iteration, but seeing if anyone is open to it.

Reply:
I think it's a good idea that we should try.